src/usbdev.c

```diff
@@ -81,6 +81,12 @@ static int usbdev_getled(struct spndev* dev);
 static inline void usbdev_parsebuttons(const struct spndev* dev, union spndev_event* evt, const unsigned char* report);
 static inline void checkrange(const struct spndev* dev, const int val);
 
+#ifdef _WIN32
+#define VID_PID_FORMAT_STR "%04x:%04x"
+#else
+#define VID_PID_FORMAT_STR "%#0.4hx:%#0.4hx"
```

Comment:
Crash? It dos not crash for me on Win32 with `"%#0.4hx:%#0.4hx"` (VS 2019).

Reply:
The crash might ahve been from when I was still trying to build it with msys2 and mingw. The ifdef is thus almost certainly too broad.